pkg/resources/schema_grant.go

```diff
@@ -142,6 +142,69 @@ func CreateSchemaGrant(d *schema.ResourceData, meta interface{}) error {
 	return ReadSchemaGrant(d, meta)
 }
 
+// UpdateSchemaGrant implements schema.UpdateFunc
+func UpdateSchemaGrant(d *schema.ResourceData, meta interface{}) error {
+	// for now the only thing we can update are roles or shares
+	// if nothing changed, nothing to update and we're done
+	if !(d.HasChange("roles") || d.HasChange("shares")) {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R005: multiple ResourceData.HasChange() calls can be combined with single HasChanges() call